### PR TITLE
Custom Module for Fileuploader

### PR DESCRIPTION
**Pull request by AlexTeusz**
This custom module opens the cognigy file-uploader plugin. you have to add the POST url and the text of the button to upload the file.

### CODE REVIEW

File: modules/cognigy-fileuploader/src/module.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    if (!args.buttonText) return Promise.reject("No upload button text defined.");
+
+    // send the message to open datepicker
+    input.actions.output("", {
+        _plugin: {
+            type: "file-uploader",
+            data: {
+                endpoint: args.endpoint,
+                uploadButtonText: args.buttonText
+
```
Comment (robinschuster) [inline modules/cognigy-fileuploader/src/module.ts]: unnecessary line break

File: modules/cognigy-fileuploader/src/module.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/**
+ * Opens the Cognigy Fileuploader Plugin
+ * @arg {CognigyScript} `endpoint` The POST endpoint to upload the file
+ * @arg {CognigyScript} `buttonText` The text of the upload button
+ */
+async function fileUploader(input: IFlowInput, args: { endpoint: string, buttonText: string }) {
```
Comment (robinschuster) [inline modules/cognigy-fileuploader/src/module.ts]: use an imperative name like "upload file"

File: modules/cognigy-fileuploader/src/module.ts (hunk 1 above)
Comment (AlexTeusz) [reply modules/cognigy-fileuploader/src/module.ts]: done

File: modules/cognigy-fileuploader/src/module.ts (hunk 2 above)
Comment (AlexTeusz) [reply modules/cognigy-fileuploader/src/module.ts]: done

**Comment by AlexTeusz** [comment]
implemented all requested changes and refactored the code. I also added tslint for testing the format. @robinschuster please check again.

**Comment by AlexTeusz** [comment]
I close the PR since it isn't used for now.

**Pull request closed without merging**